interface/src/CrashHandler_Crashpad.cpp

```diff
@@ -70,8 +76,99 @@ LONG WINAPI vectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo) {
         client->DumpAndCrash(pExceptionInfo);
     }
 
+    if (pExceptionInfo->ExceptionRecord->ExceptionCode == 0xE06D7363) {
+        fatalCxxException(pExceptionInfo);
+        client->DumpAndCrash(pExceptionInfo);
+    }
+
     return EXCEPTION_CONTINUE_SEARCH;
 }
+
+#pragma pack(push, ehdata, 4)
+
+struct PMD_internal {
```

Comment:
Where does all this come from? Please link to some documentation

Reply:
There may not be an include file for these, they're generally defined internally by the compiler -- apparently improperly though (the compiler versions use 64bit numbers that aren't compatible with the actual structures).  I did copy some of this stuff from... somewhere though

Reply:
Does that mean this is compiler specific?

Reply:
Yes this is very compiler-specific, if anyone wants to compile this for windows without using the Microsoft C++ compiler then all this code would need to be chucked.  There is a "#ifdef Q_OS_WIN" earlier in the file which I'm hoping would catch this.  I'm also making the assumption that we will not be making 32bit builds anytime soon.